deploy/deploy.sh

```
@@ -0,0 +1,293 @@
+    log_info "이전 빌드를 백업합니다..."
+    TIMESTAMP=$(date +%Y%m%d_%H%M%S)
+    cp "$PROJECT_PATH/build/libs/$JAR_NAME" "$BACKUP_DIR/${JAR_NAME}_$TIMESTAMP"
+    log_info "백업 완료: ${JAR_NAME}_$TIMESTAMP"
+  fi
+}
+
+# 프로젝트 빌드 함수
+build_application() {
+  log_info "프로젝트를 빌드합니다..."
+
+  cd $PROJECT_PATH
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)